src/AppBundle/Service/RemotePushNotificationManager.php

```diff
@@ -7,22 +7,26 @@
 use Kreait\Firebase\Factory as FirebaseFactory;
 use Kreait\Firebase\Exception\ServiceAccountDiscoveryFailed;
 use Kreait\Firebase\Messaging\CloudMessage;
+use Psr\Log\LoggerInterface;
 
 class RemotePushNotificationManager
 {
     private $firebaseFactory;
     private $apns;
     private static $enabled = true;
+    private $logger;
 
     public function __construct(
         FirebaseFactory $firebaseFactory,
         \ApnsPHP_Push $apns,
-        string $apnsCertificatePassPhrase)
+        string $apnsCertificatePassPhrase,
+        LoggerInterface $logger)
```

Comment:
Can you fix the unit tests please? 😉 
You can just inject a `NullLogger` in tests (there are examples in other tests)

Reply:
fixed